recipes/caffe2/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.8.dev" %}
+
+package:
+  name: caffe2
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/caffe2/caffe2.git
+  git_rev: master
+
+build:
+  number: 1
+  skip: True  # [win]
+
+requirements:
+  build:
+    - future
+    - glog
+    - numpy
+    - six
+    - eigen
+    - python <3
+    - protobuf ==3.4.0
+    - cmake
+    - leveldb
+    - lmdb
+    - snappy
+  run:
+    - future
+    - glog
+    - numpy
+    - six
+    - eigen
+    - python <3
+    - protobuf ==3.4.0
+    - leveldb
+    - lmdb
+    - snappy
+    - future
+
+about:
+  home: https://caffe2.ai/
+  license: BSD
```

Comment:
What version of BSD? Is it BSD 2-Clause? If so, please add that here.

Also please incl

Reply:
Also please include `license_file: LICENSE` so that we can package the license with the binaries.